AnimateQuotes.js

```diff
@@ -1,21 +1,20 @@
 import React, { useRef, useEffect, useState } from 'react';
 import { Animated, Text, View } from 'react-native';
-import { useIsShake } from './useIsShake';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_IN_DELAY, SVG_IN_DURATION, SVG_OUT_DURATION, TRANSITION_TIME_2 } from './constants';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, quotes } from './constants';
+import { getRandomInt } from './utils';
+import Quotes from "./Quotes";
 
 export default function AnimateQuotes(props) {
-  const [isRegistered, setIsRegistered] = useState(false);
-  const [hasToggledBefore, setHasToggledBefore] = useState(false);
+  const [isQuoteAnimationRunning, setIsQuoteAnimationRunning] = useState(false);
+  const [quote, setQuote] = useState(0);
```

Comment:
Moved this state from child component to parent so that setters could be called in animation wrapper's callback logic.

Reply:
```suggestion
  const [quoteIndex, setQuoteIndex] = useState(0);
```